pdc/apps/utils/__init__.py

```diff
@@ -5,7 +5,8 @@
 #
 from django.conf import settings
 
-from .messaging import DummyMessenger, KombuMessenger, FedmsgMessenger, ProtonMessenger
+from .messaging import (DummyMessenger, KombuMessenger, FedmsgMessenger,
+                        ProtonMessenger, StompMessenger)
```

Comment:
I think we won't support `proton`. It's better to remove it, also in spec, requirement and settings. Otherwise it will mislead users.